etc/kernel-launchers/kubernetes/scripts/launch_kubernetes.py

```diff
@@ -225,6 +225,8 @@ def launch_kubernetes_kernel(  # noqa
                     f"kernel launch terminating!"
                 )
         else:
+            print("ERROR processing Kubernetes yaml file - kernel launch terminating!")
+            print(k8s_yaml)
```

Comment:
Hi @lresende.  Do you have an example of what the print output looks like?  I suspect this will output the entire yaml on a single line and I wonder, since this would be used primarily for troubleshooting, if using pretty print `pprint` would be more helpful?

Reply:
It does in a pretty way, at least on the spark operator:
```
apiVersion: "sparkoperator.k8s.io/v1beta2"
kind: SparkApplication
metadata:
  name: xxxx-dafe8b37-9579-4cda-a8ab-d9f33a6dadba
  labels:
    proxyUser: "xxxxx"
spec:
  serviceAccountName: xxx
  mode: cluster
  type: Python
```

Reply:
Ok. Cool.